config/config.php

```diff
@@ -155,34 +155,34 @@
 		],
 		'14' => [
 			'100' => [
-				'latest' => '14.0.12',
-				'internalVersion' => '14.0.12.0',
-				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-14.0.12.zip',
+				'latest' => '14.0.13',
+				'internalVersion' => '14.0.13.1',
+				'downloadUrl' => 'https://download.nextcloud.com/server/releases/nextcloud-14.0.13.zip',
```

Comment:
@rullzer is there a reason why this file is not on the server?

Reply:
Fixed in the meantime.